cmd/frostfs-adm/internal/modules/morph/netmap_candidates.go

```diff
@@ -0,0 +1,62 @@
+package morph
+
+import (
+	"fmt"
+
+	netmapCore "github.com/TrueCloudLab/frostfs-node/pkg/core/netmap"
+	netmapClient "github.com/TrueCloudLab/frostfs-node/pkg/morph/client/netmap"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	sc "github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var netmapCandidatesCmd = &cobra.Command{
+	Use:   "netmap-candidates",
+	Short: "List netmap candidates nodes",
+	PreRun: func(cmd *cobra.Command, _ []string) {
+		_ = viper.BindPFlag(endpointFlag, cmd.Flags().Lookup(endpointFlag))
+		_ = viper.BindPFlag(alphabetWalletsFlag, cmd.Flags().Lookup(alphabetWalletsFlag))
+	},
+	RunE: listNetmapCandidatesNodes,
+}
+
+var short bool
+
+func init() {
+	netmapCandidatesCmd.Flags().StringP(endpointFlag, "r", "", "N3 RPC node endpoint")
+	netmapCandidatesCmd.Flags().String(alphabetWalletsFlag, "", "Path to alphabet wallets dir")
+	netmapCandidatesCmd.Flags().BoolVar(&short, "short", false, "Shortens output of node info")
+}
+
+func listNetmapCandidatesNodes(cmd *cobra.Command, _ []string) error {
+	wCtx, err := newInitializeContext(cmd, viper.GetViper())
```

Comment:
that context is usually used for complex transactions. for RO purposes it looks too heavy, see `morph dump-config` as an example

Reply:
Refactored.